plugin/driver/driver.go

```diff
@@ -296,6 +309,8 @@ func (driver *driver) joinEndpoint(w http.ResponseWriter, r *http.Request) {
 		Gateway:       gatewayIP,
 	}
 
+	AddMetaconfig(domainid, bridgeID, j.SandboxKey[22:], endID, mac[:17])
```

Comment:
SandboxKey actually has full path to that namespace, I am trimming ID part out of it.